### PR TITLE
Fix ZEDGE app

### PR DESCRIPTION
**Pull request by BlazDT**
https://forum.adguard.com/index.php?threads/fresh-install-on-moto-x4-one-specific-app-is-not-working-with-adguard.36898

### CODE REVIEW

File: Filters/exceptions.txt [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ! Once you enable private DNS, Android 9 starts resolving random domains looking like
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
+! https://forum.adguard.com/index.php?threads/fresh-install-on-moto-x4-one-specific-app-is-not-working-with-adguard.36898
+@@||api.prod.zedge.net^|
```
Comment (Alex-302) [inline Filters/exceptions.txt]: which rule and filter block it? Cannot find.

Comment (BlazDT) [reply Filters/exceptions.txt]: Screenshot_20200113-200201

Comment (BlazDT) [reply Filters/exceptions.txt]: Or better exclude that wide rule?

Comment (Alex-302) [reply Filters/exceptions.txt]: we have exclusions for similar cases(two groups of IP). Please add like existing. Cannot show at the moment:(

Comment (Alex-302) [reply Filters/exceptions.txt]: image

Comment (BlazDT) [reply Filters/exceptions.txt]: Done.

**Comment by Alex-302** [comment]
Hm. I don't see this rule in the filter. Is SDN fresh?
image

**Comment by BlazDT** [comment]
When I opened pull request it was up2date. Did manual update again and now exclusion is not required anymore.
Now likely we need to wait a while until DNS replicates filter rules.

Why are there such very wide rules in SDN filter in the first place?

**Pull request closed without merging**